app/models/url.rb

```diff
@@ -1,31 +1,9 @@
 class Url < ApplicationRecord
   include CharsArray
+  include LinksGenerator
 
   validates :url_orginal, presence: true
   validates :url_orginal, url: true
 
   before_create :generate_short_url
```

Comment:
This is an indication of a bad architecture since you're using a method defined in a module as a callback on the model therefore it's easily breakable by excluding the module